### ISSUE TITLE
Investigate select elements missing labels

### ISSUE THREAD

**Issue by agoslen** [DLFMetadataAssessment/Sandbox#344]
In our [accessibility testing](https://github.com/DLFMetadataAssessment/Sandbox/issues?q=state%3Aopen%20label%3A%222025%20accessibility%20evaluation%22) with the WAVE tool, the Tools page and the Skill-Building Resources page reported alerts related to select elements not having labels. See the following issues for more detail:

https://github.com/DLFMetadataAssessment/Sandbox/issues/341
https://github.com/DLFMetadataAssessment/Sandbox/issues/340

Please research whether adding a label to the select element would improve the accessibility of these pages. If a label is needed, implement an appropriate label on each page.
